apps/vc-api/src/vc-api/credentials/dtos/presentation.dto.ts

```diff
@@ -57,5 +58,6 @@ export class PresentationDto implements Presentation {
    * The Verifiable Credentials
    */
   @ValidateNested()
+  @Type(() => VerifiableCredentialDto)
```

Comment:
Should we also add @IsArray? Also is @ValidateNested will check verifiableCredential or each element in verifiableCredential? If I remember correctly I added `each` = true to @ValidateNested config in such cases

Reply:
I 100% agree. I made changes in bulk when fixing it. Thanks!

However, now I see that tests need to be adjusted. They fail after this change.